### PR TITLE
Sovrn Bid Adaptter: add video to Sovrn adapter

### PR DESCRIPTION
**Pull request by cpabst**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Adding support for Video to the Sovrn bid adapter

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- cpabst@sovrn.com
- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

https://github.com/prebid/prebid.github.io/pull/3492
https://github.com/prebid/prebid.github.io/pull/3493

### CODE REVIEW

File: modules/sovrnBidAdapter.js [hunk 1]
```diff
@@ -1,12 +1,38 @@
...
+  'sequence': (value) => isInteger(value),
+  'battr': (value) => Array.isArray(value) && value.every(v => v >= 1 && v <= 17),
+  'maxextended': (value) => isInteger(value),
+  'minbitrate': (value) => isInteger(value),
+  'maxbitrate': (value) => isInteger(value),
+  'boxingallowed': (value) => [0, 1].indexOf(value) !== -1,
+  'playbackmethod': (value) => Array.isArray(value) && value.every(v => v >= 1 && v <= 6),
+  'playbackend': (value) => [1, 2, 3].indexOf(value) !== -1,
+  'delivery': (value) => [1, 2, 3].indexOf(value) !== -1,
+  'pos': (value) => [0, 1, 2, 3, 4, 5, 6, 7].indexOf(value) !== -1,
```
Comment (osazos) [inline modules/sovrnBidAdapter.js:29]: for long list, you can write in the same way of the others: `value >= 0 && value <=7,`

**Review by osazos — COMMENTED** [review]
Hi @cpabst Can you update your .md file with a video example? 
Globally it is ok for me, just check my comments and adjust your code if needed.

File: modules/sovrnBidAdapter.js [hunk 2]
```diff
@@ -50,27 +76,37 @@ export const spec = {
...
         iv = iv || getBidIdParameter('iv', bid.params)
 
-        let bidSizes = (bid.mediaTypes && bid.mediaTypes.banner && bid.mediaTypes.banner.sizes) || bid.sizes
-        bidSizes = ((isArray(bidSizes) && isArray(bidSizes[0])) ? bidSizes : [bidSizes])
-        bidSizes = bidSizes.filter(size => isArray(size))
-        const processedSizes = bidSizes.map(size => ({w: parseInt(size[0], 10), h: parseInt(size[1], 10)}))
         const floorInfo = (bid.getFloor && typeof bid.getFloor === 'function') ? bid.getFloor({
           currency: 'USD',
-          mediaType: 'banner',
+          mediaType: bid.mediaTypes && bid.mediaTypes.banner ? 'banner' : 'video',
```
[the remark this answers is not in the document]
Comment (cpabst) [reply modules/sovrnBidAdapter.js:81]: It is what we expect.

**Review by osazos — APPROVED** [review]
LGTM

**Comment by maximpakhomov** [comment]
Hi @osazos .md file updated with a video example

File: modules/sovrnBidAdapter.js (hunk 1 above)
Comment (maximpakhomov) [reply modules/sovrnBidAdapter.js]: Thanks!

File: modules/sovrnBidAdapter.js [hunk 3]
```diff
@@ -50,27 +76,37 @@ export const spec = {
...
             w: 1,
             h: 1,
-          },
-          tagid: String(getBidIdParameter('tagid', bid.params)),
-          bidfloor: floorInfo.floor
+          };
+        }
+        if (
+          deepAccess(bid, 'mediaTypes.video') &&
+          deepAccess(bid, 'mediaTypes.video.context') !== ADPOD
```
[the remark this answers is not in the document]
Comment (maximpakhomov) [reply modules/sovrnBidAdapter.js]: Thanks!

**Pull request merged**